server/sonar-server/src/main/java/org/sonar/server/activity/index/ActivityIndexDefinition.java

```diff
@@ -54,7 +53,7 @@ public void define(IndexDefinitionContext context) {
 
     // type "activity"
     NewIndex.NewIndexType mapping = index.createType(TYPE);
-    mapping.setAttribute("_id", ImmutableMap.of("path", FIELD_KEY));
+    // mapping.setAttribute("_id", ImmutableMap.of("path", FIELD_KEY));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)